app/themes/dashboard/ui/styles/theme.less

```diff
@@ -43,3 +43,11 @@ z-index: 10;
 .pre-xml {
   background-color: #ecf0f5;
 }
+
+.nav-pills>li>a {
+  border-radius: 4px !important;
+}
+
+.nav-pills>li.active>a {
+  background-color: #384246 !important;
```

Comment:
This line should be removed. There are multiple themes that can be chosen and this !important will cause a lot of trouble.